certificate-analyser/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+}
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile libraries.mockito_core
+    testImplementation 'org.mockito:mockito-inline:2.13.0'
+
+    implementation 'info.picocli:picocli:4.6.1'
+    annotationProcessor 'info.picocli:picocli-codegen:4.6.1'
+}
+
+compileJava {
+    options.compilerArgs += ["-Aproject=${project.group}/${project.name}"]
+}
+
+jar {
```

Comment:
Why not publish this artifact? Or do you plan to add publishing later?

Reply:
isnt this done by publish.gradle?